public/test/helpers/alertingRuleEditor.tsx

```diff
@@ -26,6 +27,8 @@ export const ui = {
       contactPoint: byTestId('contact-point-picker'),
       routingOptions: byText(/muting, grouping and timings \(optional\)/i),
     },
+    switchModeBasic: (stepNo: number) => byTestId(`advanced-switch-${stepNo}-basic`),
+    switchModeAdvanced: (stepNo: number) => byTestId(`advanced-switch-${stepNo}-advanced`),
```

Comment:
Instead of relying on numbers, maybe it would be easier to create an enum for all steps
```ts
enum FormStep {
	Folder = 0,
	Query = 1,
	etc...
}
```
Then we wouldn't need to explain in comments what these numbers are